recipes-bsp/imx-mkimage/imx-boot_0.2.bb

```diff
@@ -180,7 +180,7 @@ do_deploy() {
         install -m 0644 ${S}/${BOOT_CONFIG_MACHINE}-${target} ${DEPLOYDIR}
     done
     cd ${DEPLOYDIR}
-    ln -sf ${BOOT_CONFIG_MACHINE}-${IMAGE_IMXBOOT_TARGET} ${BOOT_CONFIG_MACHINE}
+    ln -sf ${BOOT_CONFIG_MACHINE}-${IMAGE_IMXBOOT_TARGET} ${BOOT_NAME}
```

Comment:
I am fine with this but please split this out to another patch.

Reply:
done